python/test/unit/fem/test_interpolation.py

```diff
@@ -1025,3 +1026,111 @@ def f_test2(x):
 
     l2_error = assemble_scalar(form((u1 - u1_exact) ** 2 * dx_cell(cell_label), dtype=xtype))
     assert np.isclose(l2_error, 0.0, rtol=np.finfo(xtype).eps, atol=np.finfo(xtype).eps)
+
+
+def test_submesh_interpolation():
+    """Test interpolation of a function between a submesh and its parent"""
+    mesh = create_unit_square(MPI.COMM_WORLD, 6, 7)
+
+    def left_locator(x):
+        return x[0] <= 0.5 + 1e-14
```

Comment:
Not in house style `np.isclose`

Reply:
Same comment as below, it is all values smaller or equal to 0.5